pages/linux/links.md

```diff
@@ -0,0 +1,28 @@
+# links
+
+> Command-line web browser.
+> More information: <http://links.twibright.com/>.
+
+- Visit a website:
+
+`links {{https://example.com}}`
+
+- Apply restrictions for anonymous account:
+
+`links -anonymous {{https://example.com}}`
+
+- Enable Cookies (`1` to enable):
+
+`links -enable-cookies {{0|1}} {{https://example.com}}`
+
+- Navigate forwards and backwards through the links on a page:
+
+`Up arrow key, Down arrow key`
+
+- Go forwards and backwards one page:
+
+`Left arrow key, Right arrow key`
+
+- Exit:
+
+`q then y`
```

Comment:
```suggestion
`q then y keys`
```

Reply:
I think it'd be better to keep this page similar to the `lynx` page.

Reply:
Sure, I too think the same. I will resolve the conversation.